pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -1,47 +1,56 @@
-{ autoPatchelfHook
+{ lib
+, cmake
+, fetchFromGitHub
 , buildPythonPackage
-, fetchPypi
-, isPy39
-, lib
-, six
-, stdenv
+, abseil-cpp
+, absl-py
+, attrs
+, numpy
+, pybind11
+, wrapt
 }:
 
 buildPythonPackage rec {
   pname = "dm-tree";
-  version = "0.1.6";
-  format = "wheel";
-
-  # At the time of writing (8/19/21), there are releases for 3.6-3.9. Supporting
-  # all of them is a pain, so we focus on 3.9, the current nixpkgs python3
-  # version.
-  disabled = !isPy39;
-
-  src = fetchPypi {
-    inherit version format;
-    sha256 = "1f71dy5xa5ywa5chbdhpdf8k0w1v9cvpn3qyk8nnjm79j90la9c4";
-    pname = "dm_tree";
-    dist = "cp39";
-    python = "cp39";
-    abi = "cp39";
-    platform = "manylinux_2_24_x86_64";
+  # As of 2021-12-29, the latest stable version still builds with Bazel.
+  version = "42e87fda83278e2eb32bb55225e1d1511e77c10c";
```

Comment:
please use `unstable-YYYY-MM-DD` where the date is the date of the commit used

Reply:
Done.